### PR TITLE
Create Load Test Utility

### PR DESCRIPTION
**Pull request by parc-jason**
# Description
There is a readme included to provide details of the utility.
This has been run against DEV and has borne fruit - found a definite load issue exporting submissions.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-77

File: load-test/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "chefs-load-test",
+  "version": "1.0.0",
```
Comment (jujaga) [inline load-test/package.json:3]: Let's have this be version 0.1.0 to keep it aligned with the impending release.

Comment (parc-jason) [reply load-test/package.json:3]: Not going to coordinate this package with other package versions... no one will keep it up to date.

File: load-test/config/default.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "password": "",
+    "username": "chefs-load-test"
+  },
+  "api": {
+    "apiPath": "api/v1",
+    "basePath" : "/app",
+    "host": "https://chefs-dev.pathfinder.gov.bc.ca"
+  },
+  "submissions": {
+    "count": 1000,
```
Comment (TimCsaky) [inline load-test/config/default.json:15]: woah! this all looks awesome.. pshhh.. 
I was going to try running it locally at some point.. then was concerned i would crash something.
maybe this count should have a lower default? you mentioned 5 above

Comment (parc-jason) [reply load-test/config/default.json:15]: you can lower it in a local.json or by environment variable. you need to set the password anyway.

**Review by loneil — APPROVED** [review]
Good stuff, tested it out against my local with a small batch

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-77

**Pull request merged**